tests/unit/index-nodetest.js

```diff
@@ -70,7 +70,68 @@ describe('base plugin', function() {
       plugin.log('foo', {verbose: true});
       assert.deepEqual(verboseUi.messages, ['\u001b[34m|    \u001b[39m']);
     });
-
   });
 
+  describe('plugin helper', function() {
+    it('provides access to the oringal default values', function() {
+      var Plugin = Subject.extend({
+        requiredConfig: ['bar'],
+        defaultConfig: {
+          foo: function(context) {
+            return context.foo;
+          }
+        }
+      });
+
+      var plugin = new Plugin({
+        name: 'blah'
+      });
+
+      var context = {
+        foo: 'foo',
+        config: {
+          blah: {
+            foo: function(context, pluginHelper) {
+              return pluginHelper.readConfigDefault('foo') + 'foo';
+            },
+            bar: function(context, pluginHelper) {
+              return pluginHelper.readConfigDefault('bar') + 'bar';
```

Comment:
as bar is not defined prior on the default configuration do we want to make sure that this does not fail and returns `undefined` as expected? when reading this the first time it looked kind of strange to check for a string of "undefinedbar".

It might also be useful to add a test that static values on the default configuration work as expected as well.